config/manifests/bases/elasticsearch-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,298 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "logging.openshift.io/v1",
+          "kind": "Elasticsearch",
+          "metadata": {
+            "name": "elasticsearch"
+          },
+          "spec": {
+            "managementState": "Managed"
+            "nodeSpec": {
+              "resources": {
+                "limits": {
+                  "memory": "1Gi"
+                },
+                "requests": {
+                  "cpu": "100m"
+                  "memory": "1Gi"
+                }
+              }
+            },
+            "nodes": [
+              {
+                "nodeCount": 1,
+                "roles": ["client", "data", "master"],
+                "storage": {
+                  "size": "20G"
+                }
+              }
+            ],
+            "redundancyPolicy": "ZeroRedundancy",
+            "indexManagement": {
+              "policies": [
+                {
+                  "name": "infra-policy",
+                  "pollInterval": "30m",
+                  "phases": {
+                    "hot": {
+                      "actions": {
+                        "rollover": {
+                          "maxAge": "8h"
+                        }
+                      }
+                    },
+                    "delete": {
+                      "minAge": "2d"
+                    }
+                  }
+                }
+              ],
+              "mappings": [
+                {
+                  "name": "infra",
+                  "policyRef": "infra-policy",
+                  "aliases": ["infra", "logs.infra"]
+                }
+              ]
+            }
+          }
+        },
+        {
+          "apiVersion": "logging.openshift.io/v1",
+          "kind": "Kibana",
+          "metadata": {
+            "name": "kibana"
+          },
+          "spec": {
+            "managementState": "Managed",
+            "replicas": 1,
+            "nodeSelector": {},
+            "resources": {
+              "limits": {
+                "memory": "512Mi"
+              },
+              "requests": {
+                "memory": "512Mi"
+              }
+            }
+          }
+        }
+      ]
+    capabilities: Full Lifecycle
+    categories: OpenShift Optional, Logging & Tracing
+    certified: "false"
+    containerImage: quay.io/openshift/origin-elasticsearch-operator:latest
+    createdAt: "2020-11-04T08:00:00Z"
+    description: |
```

Comment:
We need to re-introduce the changes of #579 back in here and regenerate the CSV.

Reply:
I have updated the description at line 265 according to #579. This section seems different than that one. Shall I make them the same content?